data/core/units/humans/Loyalist_Pikeman.cfg

```diff
@@ -14,7 +14,7 @@
     advances_to=Halberdier
     cost=24
     usage=fighter
-    description= _ "Promising spearmen in the armies of Wesnoth are often graduated to wielding pikes, and are outfitted with something far superior to the motley collection of leather armor they wore as recruits. A pike is a much longer weapon than a spear, and thus facilitates different combat tactics. A wall of pikemen is the bane of any cavalry charge, and with proper discipline and tactics, pikemen can also hold most other infantry at bay."
+    description= _ "Promising spearmen in the armies of Wesnoth are often graduated to wielding pikes, and are outfitted with something far superior to the motley collection of leather armor they wore as recruits. A pike is a much longer weapon than a spear, and thus facilitates different combat tactics. A wall of Pikemen is the bane of any cavalry charge, and with proper discipline and tactics, pikemen can also hold most other infantry at bay."
```

Comment:
Second instance of pikemen not capitalised.

Also, referring to their lower-level counterparts, spearmen should be capitalised too.